docs/content/en/docs-dev/contribution-guidelines/development.md

```diff
@@ -24,15 +25,50 @@ description: >
 - `make test/web`: runs all unit tests of web.
 - `make test/integration`: runs integration tests.
 
-- `make run/pipecd`: runs Control Plane locally.
-- `make run/piped`: runs Piped Agent locally.
-- `make run/web`: runs frontend locally.
 - `make run/site`: runs PipeCD site locally (requires [hugo](https://github.com/gohugoio/hugo) with `_extended` version `0.92.1` or later to be installed).
 
 - `make gen/code`: generate Go and Typescript code from protos and mock configs. You need to run it if you modified any proto or mock definition files.
 
 For the full list of available commands, please see the Makefile at the root of repository.
 
+## How to run Piped agent locally
+
+1. Prepare the piped configuration file `piped-config.yaml`
+
+2. Ensure that your `kube-context` is connecting to the right kubernetes cluster
+
+2. Run the following command to start running `piped`
+
+    ``` console
+    make run/piped CONFIG_FILE=piped-config.yaml
+    ```
+
+## How to run Control Plane locally
```

Comment:
should we make this section before the run piped section? Since the Control Plane is required when we run-up Piped and someone who is not yet familiar with pipecd may forget to prepare Control Plane before they run up their Piped. 🤔

Reply:
Nice idea. Let me apply that.